projects/framework-blender/extensions/widgets/blender_scene_options_selector.py

```diff
@@ -0,0 +1,98 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+try:
+    from PySide6 import QtWidgets, QtCore
+except ImportError:
+    from PySide2 import QtWidgets, QtCore
+
+from ftrack_framework_qt.widgets import BaseWidget
+
+
+class BlenderSceneOptionsSelectorWidget(BaseWidget):
+    '''Widget to choose between export type and export format'''
+
+    name = 'blender_scene_options_selector'
+    ui_type = 'qt'
+
+    def __init__(
+        self,
+        event_manager,
+        client_id,
+        context_id,
+        plugin_config,
+        group_config,
+        on_set_plugin_option,
+        on_run_ui_hook,
+        parent=None,
+    ):
+        self._export_type_cb = None
+        self._extension_format_cb = None
+
+        super(BlenderSceneOptionsSelectorWidget, self).__init__(
+            event_manager,
+            client_id,
+            context_id,
+            plugin_config,
+            group_config,
+            on_set_plugin_option,
+            on_run_ui_hook,
+            parent,
+        )
+
+    def pre_build_ui(self):
+        layout = QtWidgets.QVBoxLayout()
+        layout.setContentsMargins(0, 0, 0, 0)
+        layout.setAlignment(QtCore.Qt.AlignTop)
+        self.setLayout(layout)
+        self.setSizePolicy(
+            QtWidgets.QSizePolicy.Preferred, QtWidgets.QSizePolicy.Fixed
+        )
+
+    def build_ui(self):
+        '''build function widgets.'''
+        # Create export type combo box
+        self._export_type_cb = QtWidgets.QComboBox()
+        self._export_type_cb.addItems(["scene", "selection"])
+        self.layout().addWidget(self._export_type_cb)
+
+        # Create extension format type combo box
+        self._extension_format_cb = QtWidgets.QComboBox()
+        self._extension_format_cb.addItems(["blend"])
```

Comment:
If only one item is available, I don't think this is needed at all.

Reply:
removed.

Reply:
Still in there. You can totally remove it.